unreleased.rst

```diff
@@ -33,7 +33,4 @@ HEAD — ongoing
 - [Security] Document how to verify the signature on release tarballs.
 + [DAML Ledger Integration Kit] The TTL for commands is now read from the configuration service.
 + [DAML Ledger Integration Kit] The contract key tests now live under a single test suite and are multi-node aware.
-- [DAML Compiler] **BREAKING CHANGE** Desugar template instances to ``type`` synonyms instead of ``newtype``s.
-  This type definition is easier to use in the surface DAML and consistent with the types translated to DAML LF.
-  The only change to user DAML is that uses of the ``TI`` ``newtype`` generated from
-  ``template instance TI = T A1 .. AN`` should be simplified to use the type synonym ``type TI = T A1 .. AN``.
+- [DAML Compiler] Generic template instantiations like ``template instance IouProposal = Proposal Iou`` now generate a type synonym ``type IouProposal = Proposal Iou`` that can be used in DAML. Before, they generated a ``newtype``, which cannot be used anymore.
```

Comment:
Why is this not breaking? Because we never documented the `newtype` in the surface language?